### DISCUSSION TITLE
Channel removed because it violates community guidelines

### DISCUSSION THREAD

**Discussion by sebibasti0815** [FreeTubeApp/FreeTube#6163] [General]
What kind of snowflakes determine which channels I can subscribe to with FreeTube? This can't be true anymore!

![image](https://github.com/user-attachments/assets/160bff29-b5e8-4a7d-84d5-470293376d76)

**Comment by absidue** [comment]
YouTube terminated the channel, which means it is no longer accessible anywhere, so all FreeTube can do is show you the same message you see on YouTube, because that is the only information that YouTube provides.
